tsl/src/compression/compression.c

```diff
@@ -157,6 +157,51 @@ static void row_compressor_finish(RowCompressor *row_compressor);
  ** compress_chunk **
  ********************/
 
+static void
+capture_pgclass_stats(Relation rel, int *out_pages, int *out_visible, float *out_tuples)
+{
+	Relation pg_class = table_open(RelationRelationId, RowExclusiveLock);
+	HeapTuple tuple = SearchSysCacheCopy1(RELOID, ObjectIdGetDatum(RelationGetRelid(rel)));
```

Comment:
Nit: maybe pass in just relation OID instead of `Relation`.

Reply:
fixed